README.md

```diff
@@ -25,12 +25,85 @@ startrek_token: your_token_here
 python3 -m prpr.main
 ```
 
+Также удобно создать bash alias, 
+чтобы можно было запускать из любого местоположения, например, так:
+```
+alias prpr='cd /path/to/dir/prpr/ && source venv/bin/activate && python -m prpr.main'
+```
+
+### Опции запуска
+<details>
+  <summary>Развернуть список</summary>
+
+```
+usage: main.py 
```

Comment:
Perhaps a verbatim copy of `--help` output would be easiest to maintain.

Reply:
ok